source/switching_to_live/index.html.md.erb

```diff
@@ -1,37 +1,93 @@
+Within one working day, we'll email you with:
 
-* 3D Secure is not part of the payment journey on test accounts
+- how to access your new live account
+- emergency contact details so you can tell us if there's an urgent problem, for example fraudulent payments on your account
 
-* response times of test accounts do not match live accounts, because live accounts are subject to response times of payment service providers
+Your live account will appear on the [__My services__ page](https://selfservice.payments.service.gov.uk/my-services) in the GOV.UK Pay admin tool, labelled __Live card account__.
+
+You can still use your test account.
+
+## 2. Connect your live account to your PSP
+
+Once you have a GOV.UK Pay live account, connect it to:
+
+- [Stripe](/switching_to_live/set_up_a_live_stripe_account/#connect-your-live-account-to-stripe)
+- [ePDQ](/switching_to_live/set_up_a_live_epdq_account/#connect-your-live-account-to-epdq)
+- [SmartPay](/switching_to_live/set_up_a_live_smartpay_account/#connect-your-live-account-to-smartpay)
+- [Worldpay](/switching_to_live/set_up_a_live_worldpay_account/#connect-your-live-account-to-worldpay)
+
+## 3. Choose which card types to accept
+
+Choose which card types to accept by signing into the [GOV.UK Pay admin tool](https://selfservice.payments.service.gov.uk/payment-types).
+
+If you use Worldpay, SmartPay or ePDQ, you must also make sure the same card types are selected in your PSP account.
+
+### If you use Worldpay
+
+If you use Worldpay and you want to accept American Express cards, you must contact either:
+
+- Government Banking - if your Worldpay contract is through Government Banking
+- your Worldpay account manager
+
+You must send your American Express merchant ID to Government Banking or Worldpay if you’re a non-Crown merchant.
+
+## 4. Set up 3D Secure (3DS)
+
+Sign in to your [GOV.UK Pay account settings](https://selfservice.payments.service.gov.uk/settings/) and check 3D Secure is turned on.
+
+You do not need to do anything else if you use Stripe or SmartPay.
+
+If you use Worldpay, ask your Worldpay account manager to enable 3D Secure for your merchant code.
+
+If you use ePDQ, [sign in to your ePDQ account](https://payments.epdq.co.uk/Ncol/Prod/BackOffice/login/index). Select __Configuration__, then select  __Payment methods__ and check __3D Secure__ is on.
+
+## 5. Check your live account works
+
+1. Select your live account from the [__My services__ page](https://selfservice.payments.service.gov.uk/my-services) in the GOV.UK Pay admin tool.
+
+2. Select __Settings__, then __API keys__. Generate a new live API key.
+
+3. Use the new live API key to test making a real payment on your live account, using the [GOV.UK Pay API](/making-a-payment) or a [payment link](/payment_links). You must use a real payment card. Use a small amount, because GOV.UK Pay will take the payment from the card you use.
```

Comment:
Ah, good point - although this should be covered by https://github.com/alphagov/pay-tech-docs/pull/358 (which is a change to the /making-payment page linked to here). So if we can get that published before these go-live changes, it should all fit together.

Reply:
We don't mention the minimum payout in the docs - ie. you have to have a minimum amount owed to you before your PSP will deposit it into your bank account. For Stripe it's £1 - so if a service does a test payment of 20p, it will go through and is above the min including transaction fees, but they won't actually see it deposited into their bank account. If they are trying to check the full end to end process, that might confuse them. Do we think that it's our role to clarify that here? Do you want me to review 358 now as well?

Reply:
Ah I see! Yes if you could review 358 that'd be great - let's get the details right there so we avoid splitting all the relevant info across two pages.